osbs/build/build_requestv2.py

```diff
@@ -311,118 +311,48 @@ class BuildRequestV2(BaseBuildRequest):
     """
     Wraps logic for creating build inputs
     """
-    def __init__(self, build_json_store, outer_template=None, customize_conf=None):
+    def __init__(self, osbs_api, outer_template=None, customize_conf=None, user_params=None,
+                 build_json_store=None):
         """
         :param build_json_store: str, path to directory with JSON build files
         :param outer_template: str, path to outer template JSON
         :param customize_conf: str, path to customize configuration JSON
         """
         super(BuildRequestV2, self).__init__(
-            build_json_store,
-            outer_template=outer_template or DEFAULT_OUTER_TEMPLATE
+            osbs_api,
+            outer_template=outer_template or DEFAULT_OUTER_TEMPLATE,
+            user_params=user_params,
+            build_json_store=build_json_store,
         )
+        if user_params:
+            assert isinstance(user_params, BuildUserParams)
+        else:
+            user_params = BuildUserParams()
+        self.user_params = user_params
```

Comment:
This could be done above `super().__init__()` (removing the need to set `self.user_params` again here)

Reply:
Thanks, for some reason that didn't work right in my earlier testing.